### ISSUE TITLE
A11y_Model Builder_azure_Create New Experiment_Info and relationship: Narrator not reading the dropdown control names while we are navigating.

### ISSUE THREAD

**Issue by Harika1702** [dotnet/machinelearning-modelbuilder#2652]
### GitHub Tags

#WCAG1.3.1; #A11ySev2; #A11yTCS; #A11yWCAG2.1; #DesktopApp; #Win11; #Visual Studio Client; #SH_ModelBuilder_Win32_May2023; #Info and Relationships; #Screenreader; #Narrator; #FTP; #Win32; #Benchmark;

### Actual Result
Narrator not reading the dropdown control names while we are navigating.

**Issue not reproducible with NVDA.**

### Expected Result
Narrator should announce the dropdown control names while we are navigating.

### Environment
Application Name: Model Builder
Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.6.1
Microsoft Windows 11 Enterprise Build 22624.1755
Screen reader: Narrator

### Repro Steps

1. Launch the Visual Studio with at least one of the .NET workloads.
2. Tab till enable ML.NET Model Builder (Tools -> Options -> Environment -> Preview Features) and select it.
3. Create a new .NET project or open an existing .NET project.
4. Open context menu (Right-click) of the project in the solution explorer.
5. Tab till Add and then tab till Machine Learning Model and press enter.
6. Model Builder dialog will open. Tab till Scenario tab.
7. Select Scenario screen will appear.
8. Tab till "Image Classification" and press enter.
9. Setup environment screen will appear, Tab till Azure tile and select it.
10. Tab till Setup workspace and press enter.
11. Turn on Narrator.
12. Now tab till "Subscriptions" text field and press enter.
13. Now verify that Narrator is announcing the dropdown control names while we are navigating or not.

### User Impact
Screen reader user will not be able to get the proper information regarding the dropdown items while navigating.

### Attachment
![Bug_Screen reader](https://github.com/dotnet/machinelearning-modelbuilder/assets/100270074/dfc3eb55-259e-41f3-9fc7-6bb068f156ac)

**Comment by zewditu** [comment]
@Harika1702  are you able to down this sev4 as far as readable by NVDA, this might be the issue with narrator
